h-gw-iso-l2vni-rt: docker daemon not running during test-docs

https://github.com/githedgehog/fabricator/actions/runs/25362126025/job/74364544159
```
2026-05-05T09:09:57.6624859Z Cannot connect to the Docker daemon at unix:///run/docker/docker.sock. Is the docker daemon running?
2026-05-05T09:09:57.6672957Z error: Recipe `test-docs` failed on line 81 with exit code 1
2026-05-05T09:09:57.6725892Z ##[error]Process completed with exit code 1.
```

--- a/.github/workflows/run-vlab.yaml
+++ b/.github/workflows/run-vlab.yaml
@@ -123,7 +123,7 @@ jobs:
 
       - name: Docker readiness check
         run: |
-          for i in $(seq 1 30); do
+          for i in $(seq 1 90); do
             docker info > /dev/null 2>&1 && echo "Docker ready after ${i}s" && break
             echo "Waiting for Docker... ${i}s"
             sleep 1

